processor/src/test/java/org/mapstruct/ap/test/bugs/_1140/JarMapperTest.java

```diff
@@ -0,0 +1,50 @@
+package org.mapstruct.ap.test.bugs._1140;
+
+import org.mapstruct.ap.testutil.ProcessorTest;
+import org.mapstruct.ap.testutil.WithClasses;
+import org.mapstruct.ap.testutil.compilation.annotation.CompilationResult;
+import org.mapstruct.ap.testutil.compilation.annotation.Diagnostic;
+import org.mapstruct.ap.testutil.compilation.annotation.ExpectedCompilationOutcome;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+class JarMapperTest {
```

Comment:
The assertions in the test are irrelevant. What we want to test here is only the messages we have during compilation. So only `@ExpectedCompilationOutcome` for this